src/rulesets/oas3/index.json

```diff
@@ -83,7 +83,20 @@
       "message": "{{error}}",
       "recommended": true,
       "type": "validation",
-      "given": "$..[?(@.example)]",
+      "given": "$..[?(@.example && !@.type)]",
```

Comment:
Once we are at it, is there any way to make the json path expression more specific?
I understand `$..[?(@.example && !@.type)]` feels very handy, yet it's utterly slow to the point where we have that rule turned off in Studio.
The other issue I see with that expression is the use of function filter expression that accesses the properties (`@.example && @.type`). This will pretty much impossible to support when https://github.com/stoplightio/spectral/issues/437 becomes a thing.

If there is no way to make it less general, I'm okay with leaving it as is.

Reply:
Yeah I was trying to get more specific and less greedy rules like intentionally pretending with “parameters” so we know we’re looking st parameters and can make the rules more specific seeing as different types of parameters may or may not have a schema keyword, etc.